src/database/firestore.spec.ts

```diff
@@ -200,6 +304,38 @@ describe('firestore', () => {
                 );
         });
 
+        it('removes uid before storing', done => {
+            const documents: any[] = [
+                {
+                    field: '0',
+                },
+                {
+                    field: '1',
+                },
+                {
+                    field: '2',
+                }
+            ]
+
+            const documentsWithIds = documents.map(x => {
+                x = Object.assign({}, x);
```

Comment:
in case you were wondering the syntactic sugar way to do this it is the following:
```
x = {
  ...x
};